### PR TITLE
Update sync function paragraph

### PR DESCRIPTION
**Pull request by jamiltz**
@adamcfraser Can you take a look at this change. I remember chatting with @ajres  about the fact that requireAccess, requireUser and requireRole always succeed in a resync operation. I can't think of a scenario where that would have an impact. Referring to this sentence in the change.

> Keep in mind that it's perfectly fine if the Sync Function in a resync operation does not ressemble the Sync Function you expect to use in production. The former is only an intermediary function used in the resync operation and the latter is used to process reads and writes in a production environment.

### CODE REVIEW

File: md-docs/ready/guides/sync-gateway/sync-function-api.md [hunk 1]
```diff
@@ -60,13 +60,21 @@ In plain English: by default, a document will be assigned to the channels listed
...
 
 If you need to ensure access to the database during the update, you can create a read-only backup of the Sync Gateway's bucket beforehand, then run a secondary Sync Gateway on the backup bucket, in read-only mode. After the update is complete, switch to the main Gateway and bucket.
 
-In a clustered environment with multiple Sync Gateways sharing the load, all the Gateways need to share the same configuration, so they all need to be taken down together. After the configuration is updated, one instance should be brought up so it can update the database — if more than one is running at this time, they'll conflict with each other. After the first instance finishes opening the database, the others can be started.
+In a clustered environment with multiple Sync Gateways sharing the load, all the Gateways need to share the same configuration, so they all need to be taken down together. After the configuration is updated, **one** instance should be brought up so it can update the database — if more than one is running at this time, they'll conflict with each other. After the first instance finishes opening the database, the others can be started.
```
Comment (jamiltz) [inline md-docs/ready/guides/sync-gateway/sync-function-api.md]: updated the paragraph https://github.com/couchbaselabs/couchbase-mobile-portal/pull/475/commits/ccb3569c550b8a757b0c72d1320b73336c3eb4f2#diff-29aba19f3d76a4140b1970e96b672c5cR83

Did you want to also remove the following sentence?

> After the configuration is updated, **one** instance should be brought up so it can update the database — if more than one is running at this time, they'll conflict with each other. After the first instance finishes opening the database, the others can be started.

**Comment by jamiltz** [comment]
@adamcfraser updates for all your comments pushed in the second commit.
This is ready for you to merge.

**Comment by jamiltz** [comment]
@adamcfraser ping

**Comment by jamiltz** [comment]
@adamcfraser should have been clearer. The only change I need your thumbs up on is https://github.com/couchbaselabs/couchbase-mobile-portal/pull/475#discussion_r105473305.

**Comment by adamcfraser** [comment]
This looks good to me

**Comment by jamiltz** [comment]
LGTM
